Src/FluentAssertions/Formatting/XAttributeValueFormatter.cs

```diff
@@ -1,3 +1,4 @@
+using System.Collections.Generic;
```

Comment:
delete (in multiple of the Formatters)

Reply:
Because Rider collapses the `using` statements by default, it started to care less about them. But cleaned them up for you ;-)